scope

```diff
@@ -22,6 +22,13 @@ IP_ADDR_CMD="find /sys/class/net -type l | xargs -n1 basename | grep -vE 'docker
     xargs -n1 ip addr show | grep inet | awk '{ print \$2 }' | grep -oE '$IP_REGEXP'"
 WEAVESCOPE_DOCKER_ARGS=${WEAVESCOPE_DOCKER_ARGS:-}
 
+# When docker daemon is running with User Namespace enabled, this tool will run into errors:
+#  "Privileged mode is incompatible with user namespaces" for `docker run --privileged`
+#  "Cannot share the host's network namespace when user namespaces are enabled" for `docker run --net=host`
+# To avoid above errors, use `--userns=host` option to let container use host User Namespace.
+# This option(saved in $USERNS_HOST) will be inserted ONLY IF docker support `--userns` option.
+docker run --help |grep -q -- --userns && USERNS_HOST="--userns=host"
```

Comment:
Could you add a space after `|`:
```diff
-docker run --help |grep -q -- --userns && USERNS_HOST="--userns=host"
+docker run --help | grep -q -- --userns && USERNS_HOST="--userns=host"
```

lint on CircleCI would complain otherwise.

The following command would also fix it automatically:
```
shfmt -i 4 -w scope
```